.eleventy.js

```diff
@@ -69,6 +69,31 @@ module.exports = function(eleventyConfig) {
         return (tags || []).filter(tag => ["homepage"].indexOf(tag) === -1);
     });
 
+    eleventyConfig.addCollection("homepageLinks", function(collectionApi) {
+      return collectionApi.getFilteredByGlob([ 
+        "**/patterns.md",
+        "**/principles.md",
+        "**/standards.md"]);
+    });
+
+    eleventyConfig.addCollection("getAllStandards", function(collectionApi) {
+      return collectionApi.getFilteredByGlob("**/standards/*.md").sort(function(a, b) {
+          return a.data.id.localeCompare(b.data.id); // sort by ID ascending
```

Comment:
```suggestion
          return a.data.id.localeCompare(b.data.title); // sort by title ascending
```
Are we expecting most people to reference standards by ID? Think this list will be easier to browse alphabetically but happy to be challenged on that.

Reply:
I am unsure how most will reference standards. However, we are showing standards with "ID - Title" so based on that I think not ordering by ID may be confusing (doesn't look right to me, personally). My preference is ID ordering here, unless we changed the display so it did not start with ID.

Reply:
<img width="49%" style="float:left;" alt="Screenshot 2023-07-28 at 15 39 09" src="https://github.com/HO-CTO/engineering-guidance-and-standards/assets/87995501/c6ebed18-cba7-4496-8af6-bf5044c34351">
<img width="49%" style="float:left;" alt="Screenshot 2023-07-28 at 15 38 58" src="https://github.com/HO-CTO/engineering-guidance-and-standards/assets/87995501/727fa0d3-3507-4bd1-acbf-df26c101dec1">

I think the most prominent in both cases for me is the title, even if the ID is first. I found it jarring coming to the standards collection page from the other two that were in title order.

Reply:
Ah, I was referring to the sidebar links:
![image](https://github.com/HO-CTO/engineering-guidance-and-standards/assets/133027753/5b981b70-e404-4cee-8c1b-1d012914a869)

Agree it is not the best on the standards page itself. These both use the same collection, but we can make two collections for standards each with ordering on ID and title and use them both.

Reply:
I was actually going to suggest we sort by ID descending..

I think ultimately we will want to create a way people can browse the standards by tag, maybe with a good filter. So happy for us to leave as-is for now